.editorconfig

```diff
@@ -1,16 +1,7 @@
-# editorconfig.org
-
 root = true
 
 [*]
-charset = utf-8
-indent_size = 2
 indent_style = space
-insert_final_newline = true
```

Comment:
Yeah my bad, I copied a .editorconfig that didn't have it in there and GH tracked it as a deletion